extensions/amp-analytics/0.1/vendors.js

```diff
@@ -141,6 +141,35 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
     },
   },
 
+  'alexametrics': {
+    'requests': {
+      'base': 'https://${ampAtrkHost}/atrk.gif?account=${atrk_acct}&domain=${domain}', 
+      'pageview': '${base}&jsv=amp-${ampVersion}' + 
+        '&frame_height=${viewportHeight}&frame_width=${viewportWidth}' + 
+        '&title=${title}&time=${timestamp}&time_zone_offset=${timezone}' + 
+        '&screen_params=${screenWidth}x${screenHeight}x${screenColorDepth}' + 
+        '&ref_url=${documentReferrer}&host_url=${sourceUrl}&random_number=${random}' + 
+        '&user_cookie=${clientId(__auc)}&user_cookie_flag=0' + 
+        '&user_lang=${browserLanguage}&amp_doc_url=${ampdocUrl}'
+    },
+    'vars': {
+        'atrk_acct': 'xxxxxxxx'
```

Comment:
One question: do you want to keep `atrk_acct` and `domain` default value to `xxxxxxx`? Otherwise would assigning default value to `''` be better?

Reply:
I saw other vendors use xxxxxxx as the default value and include that in vendor-requests.json for testing. I actually don't mind updating it to ''.